### ISSUE TITLE
Clarify provider users versus members in group enrollments

### ISSUE THREAD

**Issue by brainwane** [OpenTechStrategies/psm#424]
In [a review comment](https://github.com/OpenTechStrategies/psm/pull/382#discussion_r135879333) about group providers adding or removing members from their enrollments, @kfogel was concerned. The relevant part of the documentation:

> _How can I update an existing organizational enrollment to add a new provider (e.g., if a clinic hires a new physician)?_

> If an enrollment is a draft (you haven't submitted it yet), then yes, you can click on the draft enrollment and edit the member list.

> If you have already submitted the enrollment, then you should have the individual, or a service agent, create a new enrollment for an individual provider. On the "Practice Info" screen, the user should say "Yes" to the question "Are you employed and/or independently contracted by a group practice?" and enter the organization's information.

Karl says:

> This sounds a little weird, in terms of the eventual results it would produce in the PSM's data. In the first scenario, it sounds like a given provider just gets another member (but there's still only one provider). In the second case, it sounds like a new provider would be created? That's an odd thing to have be dependent on the mere timing of when a doctor joined a hospital or whatever. Is this an acknowledged bug in the current PSM, and if so, can we say so and link to the appropriate issue?

> Or am I just misunderstanding the end result?

**Comment by cecilia-donnelly** [comment]
Agreed, this is complex.  I don't think what @kfogel suggested is the real situation.  I believe that the "members" of the enrollment will each need to be enrolled individually either way -- so, it's not the case that a new provider would only be created if they're added after the group provider was screened.

To be more verbose, there's something that has come up with states about "billing" versus "rendering" providers, which is really what's going on here (I think).  The group provider is the "billing" provider, and the individuals (members) are the ones "rendering" the services.  That is, when you go to a hospital, it's not as if the hospital administration / the institution writ large provides you any services.  Some individual does, and is in charge of your case.  That rendering provider needs to be registered with Medicaid even though s/he will not be the one filing the forms to get reimbursement from Medicaid.  Does that make sense?  I'm certainly not an expert on how states handle this, but it makes sense to me that even though the rendering provider doesn't bill directly, states would want to make sure that the renderer was indeed qualified to provide those services --  that is, that s/he'd been screened.  The group provider is also registered, so that it can do all the billing work, and as a biller it also needs to be screened.

So there *is* an issue here, about how we clarify and differentiate between billing and rendering providers.  We need to have more conversations with states about this, and we need to clarify how it is presented in the PSM.

**Comment by cecilia-donnelly** [comment]
Would love to hear if that makes sense, @kfogel, and if it addresses your concern!  (I know you were asking for clarification in the original comment, so let me know if this helps.)

**Comment by cecilia-donnelly** [comment]
@chj124, I believe I said I would tag you in this issue during our discussion last week about billing vs. rendering providers.  

Generally, more notes about this:

The process varies by state.  In some states, it is more or less what I've described above: individuals enroll and then are linked to a group provider.  In others, the group provider can enroll associated individuals (staff).  The PSM needs to be able to handle the latter case, and clarify the former in the UI.

**Comment by chj124** [comment]
from my understanding...

Each provider needs to be screened separately
Provider has specialties they can perform
Each specialty has an associated risk
Each specialty/risk assessment determines the approval requirements
Each provider may provide services at more than one facility/practice
Each facility/practice has specialties that may be performed there
Each facility may have different requirements for approval for each specialty
A provider may perform different specialties at different facilities
A facility/group may enter multiple providers but they will need to be screened individually based on criteria
A provider may be approved to perform a specialty at one facility but not at another
A provider may be available at multiple facilities but only approved at one
A provider may have their own direct billing.
A provider may have a common billing group within a facility, across all facilities.

A provider may have their own practice and perform specialty services at a hospital. There is the possibility to have multiple billing depending on the services provided.

An anesthesiologist may perform services at multiple facilities but their billing almost always comes from one source, separate from each facility.

Approval for a provider is 1) that they are licensed and approved to do business in the municipality and 2) that they have been screened and approved to perform a specialty at a facility.

![provider class diag](https://user-images.githubusercontent.com/5740678/31088798-ec00f5ae-a76f-11e7-9c4c-3b26ffe7647a.png)

can i get an ERD of the PSM data?
